cmd/cluster-agent-cloudfoundry/subcommands/run/command.go

```diff
@@ -67,10 +68,11 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				forwarder.Bundle,
 				fx.Provide(defaultforwarder.NewParamsWithResolvers),
 				demultiplexer.Module,
+				orchestratorForwarderImpl.Module,
+				fx.Supply(orchestratorForwarderImpl.Params{UseOrchestratorForwarder: false}),
```

Comment:
Do we need this line ? Would `UseOrchestratorForwarder` be false by default ?

Reply:
`orchestratorForwarderImpl.NewParamsDisabled` maybe ?

Reply:
Code updated.